students/2/modern-js.js

```diff
@@ -25,11 +25,12 @@ function getData(tableName) {
   return retval;
 }
 
-function main() {
-  var table = "person";
-  var personList = getData(table);
-  personList.sort(function(a,b){return a.age - b.age});
+const main = () => {
+  let table = "person";
+  let personList = getData(table);
+  personList.sort((a,b) => {return a.age - b.age});
```

Comment:
Can we simplify the sort callback (does the return and curly brace need to be here?)